<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameChoiceItem</class>
 <widget class="QWidget" name="GameChoiceItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>50</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>50</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lb_fubiaoLeft">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_fubiaoRight">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="MyButtonGameChoice" name="pb_gameChoice1Item">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>250</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>OO</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyButtonGameChoice</class>
   <extends>QPushButton</extends>
   <header>mybuttongamechoice.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
